### DISCUSSION TITLE
Grid View log button issue

### DISCUSSION THREAD

**Discussion by cho6646** [apache/airflow#29652] [Q&A]
Hi, I'm currently using airflow 2.5.0

Once i go into grid view (which works great, thank you) then click 'Logs' tab next to 'Details' tab, the browser fails to render the table with logs and the whole grid view page disappear.
<img width="920" alt="image" src="https://user-images.githubusercontent.com/20199350/220306050-982e5098-27a8-4f4a-891c-bcb74d84e260.png">
<img width="1719" alt="image" src="https://user-images.githubusercontent.com/20199350/220306174-4f2afca7-589e-4a86-abf8-0e8e1bcb3293.png">

in the browser console, it says it failed with error message (e.split is not a function) 
<img width="417" alt="image" src="https://user-images.githubusercontent.com/20199350/220306441-036ee5b5-53d9-42fc-99aa-d2ca1d381373.png">

The similar thing happened when i used grid view in airflow 2.4.2 (the whole grid view page disappeared when i click a dag box in the grid). The same error message printed as well.
However the error with grid view in airflow 2.4.2 disappeared in airflow 2.5.0.
So i tried to upgrade the airflow to 2.5.1; however the same issue happened

Does anyone have the similar issues like mine? Or have any idea resolving this?

Thank you for checking this thread! Hope to find any solution...

**Comment by potiuk** [comment]
I think you should upgrade and check it in 2.5.1 and if it stlill there, you should raise a regular issue about it. There were few UI /Grid view fixes there, I think.

**Reply by cho6646** [reply]
Already checked with 2.5.1 and the same thing happened…
Were checking if our log is wrong first

**Comment by bbovenzi** [comment]
1. In the browser's network tab, do you see the request to get logs and is it successful or is there anything strange with the response?
2. Does the regular logs page render correctly?

**Reply by bbovenzi** [reply]
Ok, the issue is with how we are parsing the logs to give us some filters in the UI. Could you possibly share the format of your logs (with sensitive data replaced with mock values)? Then I can write some tests and open a PR to fix this for 2.5.2

**Reply by cho6646** [reply]
1. it says get logs is successful and i can get the correct response with logs
2. i dont really get this one...
our log format is like
`*** Reading local file: /opt/airflow/logs/dag_id=xxxx/run_id=xxxx/task_id=xxxx/attempt=1.log\\n[2023-03-02-T00:00:00.000+0000] {file.py:0000} some log...\\n[2023-03-02-T00:00:00.000+0000] {file.py:0000} some log...\\n[2023-03-02-T00:00:00.000+0000] {file.py:0000} some log...`
